help/about-mp.txt

```diff
@@ -0,0 +1,22 @@
+ <- @@index|<<Back to Index>>@@
```

Comment:
This file should not exist, instead rely on the overlayed one from manaplus itself